Mixpanel/app_extension_module.modulemap

```diff
@@ -1,5 +1,5 @@
 framework module MixpanelAppExtension {
-    umbrella header "Mixpanel.h"
+    header "Mixpanel.h"
```

Comment:
umbrella headers are supposed to include everything in the directory (and maybe all subdirectories? not as sure about that). it was giving us a ton of warnings that it wasn't including any of the headers that we didn't want to be public. i think umbrella headers are supposed to be used if you split your folder structure into public and private files